roles/ansible_service_broker/tasks/generate_certs.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- set_fact:
+    ansible_service_broker_generate_certs: "{{ ansible_service_broker_certs_dir is undefined }}"
+
+- name: Create ansible-service-broker cert directory
```

Comment:
instead of creating a fact and having a when for each of these can you move these into a block instead with a single when?
```yaml
- when: ansible_service_broker_certs_dir is undefined
  block:
    - file: 
 <etc>
```

Reply:
Good call, that is way better